src/graphql/user/Resolvers.js

```diff
@@ -0,0 +1,73 @@
+const LOG = require('../../utils/Log');
+const connection = require('../../db');
+
+const Resolvers = {
+    Query: {
+        async getConfig(root, params) {
+            const query = {
+                text: "SELECT configuration FROM user_config WHERE username=$1 AND tenant=$2;",
```

Comment:
I think It's easy to do a sql injection here.

Reply:
What do you mean by SQL Injection? A raw string as a query?

Reply:
Sorry, It's fine :))